<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LVADataSetDialog</class>
 <widget class="QDialog" name="LVADataSetDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>251</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lblCaption">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Locally varying anisotropy for</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Windowing of</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="4">
       <widget class="QLabel" name="lblGridName">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0" rowspan="3" colspan="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>divide grid into</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QSpinBox" name="spinDivideU">
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>parts along U direction</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4" rowspan="3">
       <widget class="QPlainTextEdit" name="txtEdWindowingSummary"/>
      </item>
      <item row="2" column="2">
       <widget class="QSpinBox" name="spinDivideV">
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>parts along V direction</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QSpinBox" name="spinDivideW">
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>parts along W direction</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Type</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="radio2D">
        <property name="toolTip">
         <string>Point set will contain azimuth, semi-major and semi-minor axes.</string>
        </property>
        <property name="text">
         <string>2D (areal)</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radio3D">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Point set will contain all three axes and all three angles that defines an anisotropy ellipsoid.</string>
        </property>
        <property name="text">
         <string>3D</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioVertical">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Point set will contain only the variographic range along vertical.</string>
        </property>
        <property name="text">
         <string>1D (vertical)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btnCompute">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/icons32/play32</normaloff>:/icons32/play32</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnDismiss">
       <property name="text">
        <string>Dismiss</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnDismiss</sender>
   <signal>clicked()</signal>
   <receiver>LVADataSetDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>308</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>379</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onComputeLVA()</slot>
 </slots>
</ui>
